src/config.py

```diff
@@ -133,6 +133,8 @@ class FunctionConfig(BaseModel):
     overwrite: bool
     remove_only: bool = False
     deploy_wait_time_sec: int = 1200  # 20 minutes
+    cpu: float = 0.25
+    memory: float = 1.0
```

Comment:
```suggestion
    cpu: float = None
    memory: float = None
```

Reply:
Used Optional[float] instead like it is used for some other values. Hopefully gives the same outcome

Reply:
Your choice is better! (They do the exact same, but since we are already using Optional, it makes more sense!)